src/components/play/services/scoring/ChooseSeveralScoring.ts

```diff
@@ -6,12 +6,26 @@ const markLiveChoices = (component: any, attempt: ComponentAttempt<ChooseSeveral
 
     for (let [index, choice] of choices.entries()) {
         const checked = attempt.answer.findIndex(a => a.shuffleIndex === index) >= 0;
-        if (checked === choice.checked || (checked === false && !choice.checked)) {
-            attempt.marks += 1;
-        } else {
+        if (choice.checked){
+            attempt.maxMarks += 2;
+        }
+
+        if (checked === choice.checked && choice.checked) {
```

Comment:
checked && choice.checked reads better (both statements have to be true for condition to be true)

Reply:
maybe a comment would help in general here.